### PR TITLE
Update install.md

### PR DESCRIPTION
**Pull request by justoverclockl**
updated italian docs

### CODE REVIEW

**Review by davwheat — APPROVED** [review]

**Comment by datitisev** [comment]
Do we want to remove the `.`? Is that the default install directory?

**Comment by davwheat** [comment]
Good spot!

**Review by datitisev — CHANGES_REQUESTED** [review]
Don't remove the `.` relative path.

**Comment by justoverclockl** [comment]
i have to do something? 👀

File: docs/it/install.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Questo non significa necessariamente che tu abbia bisogno di un VPS. Alcuni host
 Flarum usa [Composer](https://getcomposer.org) per gestire le sue dipendenze ed estensioni. Prima di installare Flarum, sar� necessario [installare Composer](https://getcomposer.org) sulla tua macchina. Successivamente, esegui questo comando in una cartella vuota in cui desideri installare Flarum:
 
 ```bash
-composer create-project flarum/flarum . --stability=beta
+composer create-project flarum/flarum
```
Comment (davwheat) [inline docs/it/install.md:39]:
```suggestion
composer create-project flarum/flarum .
```

**Review by davwheat — APPROVED** [review]

**Review by luceos — APPROVED** [review]

**Pull request merged**